### ISSUE TITLE
aci_vmm_domain: unknown attribute 'descr' in element 'vmmDomP'

### ISSUE THREAD

**Issue by FrederikSuijs** [CiscoDevNet/terraform-provider-aci#294]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version
```
terraform version
Terraform v0.14.7
+ provider registry.terraform.io/ciscodevnet/aci v0.5.4
```

### APIC version and APIC Platform

* v5.1 and all

### Affected Resource(s)

* aci_vmm_domain

### Terraform Configuration Files

```hcl
resource "aci_vmm_domain" "vmm_domain_vds_01" {
  access_mode         = "read-write"
  ave_time_out        = "30"
  config_infra_pg     = "no"
  ctrl_knob           = "epDpVerify"
  enable_ave          = "no"
  enable_tag          = "no"
  encap_mode          = "unknown"
  enf_pref            = "hw"
  ep_inventory_type   = "on-link"
  ep_ret_time         = "0"
  hv_avail_monitor    = "no"
  mcast_addr          = "0.0.0.0"
  mode                = "default"
  name                = "vds_01"
  pref_encap_mode     = "unspecified"
  provider_profile_dn = "uni/vmmp-VMware"
```

### Debug Output

```
terraform state show aci_vmm_domain.vmm_domain_vds_01
# aci_vmm_domain.vmm_domain_vds_01:
```
```hcl
resource "aci_vmm_domain" "vmm_domain_vds_01" {
    access_mode               = "read-write"
    annotation                = "orchestrator:terraform"
    ave_time_out              = "30"
    config_infra_pg           = "no"
    ctrl_knob                 = "epDpVerify"
    description               = jsonencode({})
    enable_ave                = "no"
    enable_tag                = "no"
    encap_mode                = "unknown"
    enf_pref                  = "hw"
    ep_inventory_type         = "on-link"
    ep_ret_time               = "0"
    hv_avail_monitor          = "no"
    id                        = "uni/vmmp-VMware/dom-vds_01"
    mcast_addr                = "0.0.0.0"
    mode                      = "default"
    name                      = "vds_01"
    pref_encap_mode           = "unspecified"
    provider_profile_dn       = "uni/vmmp-VMware"
}
```
### Panic Output

Error: unknown attribute 'descr' in element 'vmmDomP'

  on vmm_domain.tf line 1, in resource "aci_vmm_domain" "vmm_domain_vds_01":
   1: resource "aci_vmm_domain" "vmm_domain_vds_01" {

### Expected Behavior

vmm_domain is properly created

### Actual Behavior

Terraform module crashes.
After investigating the POST, it seems after re-applying, the _descr_ field is added and tried to be pushed.
```json
{
    "vmmDomP": {
        "attributes": {
            "accessMode": "read-write",
            "annotation": "orchestrator:terraform",
            "aveTimeOut": "30",
            "configInfraPg": "no",
            "ctrlKnob": "epDpVerify",
            "descr": "",
            "dn": "uni/vmmp-VMware/dom-vds_01",
            "enableAVE": "no",
            "enableTag": "no",
            "encapMode": "unknown",
            "enfPref": "hw",
            "epInventoryType": "on-link",
            "epRetTime": "0",
            "hvAvailMonitor": "no",
            "mcastAddr": "0.0.0.0",
            "mode": "default",
            "prefEncapMode": "unspecified",
            "rn": "dom-vds_01",
            "status": "modified"
        }
    }
}
```

The API doesn't allow this.
I think the behavior is very similar to #149 

### Steps to Reproduce

Just try to apply the same plan twice should produce the error.

1. `terraform apply`
2. change something in the resource/plan; e.g. _relation_infra_rs_vlan_ns_
3. `terraform apply`

### References

* #149

**Comment by anvitha-jain** [comment]
This issue was due to a discrepancy in the [vmmDomP] class, which has been resolved in the PR associated with issue #175.  The solution will be updated with the next release.

**Comment by lhercot** [comment]
Fixed in release v0.6.0. Can you check if the fix works for you?

**Comment by wietersr** [comment]
I'm getting same Error now "unknown attribute 'descr' in element 'vmmCtrlrP" using >=0.7.0 provider
